locust/templates/report.html

```diff
@@ -39,7 +39,8 @@
 </head>
 <body>
     <div class="container">
-        <h1>Locust Test Report</h1>
+        <h1>{{ escape(str(body_title)) }}</h1>
+        {%if body_subtitle is not none%}<h2>{{ escape(str(body_subtitle)) }}</h2>{% endif %}
```

Comment:
I think having an empty (`<h2></h2>`) block doesnt change how it looks in the browser, so no need for the if-statement?

Reply:
Yes! It is needed. For the case it is None.